gengokit/httptransport/httptransport.go

```diff
@@ -12,15 +12,14 @@ import (
 	"unicode"
 
 	log "github.com/Sirupsen/logrus"
-	"github.com/TuneLab/go-truss/deftree"
-	"github.com/TuneLab/go-truss/gengokit/clientarggen"
+	"github.com/TuneLab/go-truss/svcdef"
 	gogen "github.com/golang/protobuf/protoc-gen-go/generator"
 	"github.com/pkg/errors"
 )
 
 // Helper is the base struct for the data structure containing all the
 // information necessary to correctly template the HTTP transport functionality
-// of a microservice. Helper must be built from a deftree.
+// of a microservice. Helper must be built from a Svcdef.
```

Comment:
I think we should follow what Ian said and try to switch to the word "service" rather than "microservice". It has less baggage from being a buzz word. I have been changing them when I run across them. 

If you make more modifications to this pull, consider changing it here.